_includes/tutorials/tumbling-windows/flinksql/code/src/test/java/io/confluent/developer/FlinkSqlTumblingWindowTest.java

```diff
@@ -18,6 +18,7 @@ public void testTumblingWindows() throws Exception {
     streamTableEnv.executeSql(getResourceFileContents("populate-ratings.sql"));
     streamTableEnv.executeSql(getResourceFileContents("create-ratings-by-6hr-window.sql.template",
```

Comment:
now that line 19 is just the CREATE TABLE statement, tack on `.await()` that should prevent race